src/views/InteractiveMap/Escher.vue

```diff
@@ -349,6 +360,37 @@ export default Vue.extend({
       });
       return fluxesFiltered;
     },
+    ecModelFluxes(fluxes) {
+      // Map fluxes from an enzyme-constrained model to the corresponding
+      // escher map, as reaction ids from ecModels have additional prefixes
+      // and/or suffixes.
+      const fluxesMapped = {};
+      Object.keys(fluxes).forEach(rxn => {
+        var newRxn = rxn;
+        var newFlux = fluxes[rxn];
+        if (rxn.startsWith("arm_")) {
+          // For isozymes, use the flux in the reaction "arm_XXX".
+          newRxn = rxn.substring(4);
+        } else if (rxn.endsWith("No1")) {
+          // For single enzymes, use the flux in the reaction "XXXNo1" (for
+          // this, check first that no arm reaction is already present).
+          const rootRxn = rxn.substring(0, rxn.length - 3);
+          if (!Object.keys(fluxesMapped).includes(rootRxn)) {
```

Comment:
```suggestion
          if (!(rootRxn in fluxesMapped)) {
```

Reply:
If it is already present, you keep `XXXNo1: flux` as is?

Reply:
yes, because it would mean that `arm_XXX` was already added as `XXX`

Reply:
Okay

If you want code looking more like python version of same code:
```py
#Single enzyme:
elif rxn_id.endswith("No1") and rxn_id[:-3] not in parsed_fluxes.index:
    new_id = rxn_id[:-3]
```

```ts
} else if (rxn.endsWith("No1") && !(rxn.slice(0, -3) in fluxesMapped)) {
  // For single enzymes, use the flux in the reaction "XXXNo1" (for
  // this, check first that no arm reaction is already present).
  newRxn = rxn.slice(0, -3)
}
```

Reply:
> If you want code looking more like python version of same code:

done in [`e439c4b`](https://github.com/DD-DeCaF/caffeine-vue/pull/171/commits/e439c4bcd966bf86533764282d8b2e3fae41c20c)